packages/card/src/Card.js

```diff
@@ -43,6 +53,22 @@ const Root = createStyledComponent(
 /**
  * Card component
  */
-export default function Card(props: Props) {
-  return <Root {...props} />;
+export default class Card extends Component {
+  props: Props;
+
+  render() {
+    const rootProps = {
+      onKeyPress: this.props.onClick ? this._onKeyPress.bind(this) : undefined,
+      tabIndex: this.props.onClick ? 0 : undefined,
+      ...this.props
```

Comment:
How does this affect a11y?  Should we add `role="button"` when `onClick` is present?